go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 	gopkg.in/yaml.v3 v3.0.1
 )
+
+replace github.com/canonical/x-go => github.com/rebornplusplus/canonical-x-go v0.0.0-20230425082201-61c9f268b5f0
```

Comment:
This needs to be dropped.

Reply:
Dropped it, thanks. Please note that the tests are failing since `shlex.Join()` is missing. I will update Line 7 (quoted below) as soon as https://github.com/canonical/x-go/pull/19 gets merged.
```go
	github.com/canonical/x-go v0.0.0-20230113154138-0ccdb0b57a43
```

Reply:
Updated to the new version in https://github.com/canonical/pebble/pull/191/commits/43ef4ef1b60f77561f8da9b016fdf81f412f34b3.